src/librustc_back/target/armv7_unknown_linux_gnueabihf.rs

```diff
@@ -23,7 +23,8 @@ pub fn target() -> TargetResult {
         target_vendor: "unknown".to_string(),
 
         options: TargetOptions {
-            features: "+v7,+vfp3,+neon".to_string(),
+            // Info about features at https://wiki.debian.org/ArmHardFloatPort
+            features: "+v7,+vfp3,+d16".to_string(),
             cpu: "cortex-a8".to_string(),
```

Comment:
Hm yes it appears to still enable NEON! I'm not sure what cpu to use otherwise, so I've fallen back to "generic" for now and we can rely on features to enable bits I believe.